### ISSUE TITLE
Record access control is unpredictable for newly created ontologies

### ISSUE THREAD

**Issue by gregid** [inovexcorp/mobi#10]
[All done with admin account]

First ontology I created in fresh mobi 1.15.57 installation had most of the functionality disabled. I was only able to Create new tags, rest of + options were greyed out.

Second uploaded ontology gave me possibility to add but existing objects in the list are greyed out - I can select/view them but can't edit. (Problem persists on created branch)

Third created ontology - I have all access to add/edit/delete

I made sure the problematic ontologies have Record Access Control set to Everyone for all operations - no effect

**Comment by mnmercer** [comment]
Hi @gregid . I just downloaded 1.15.57 from our website, created a brand new ontology, and had complete edit functionality.

In the first case, it sounds like you are looking at a specific Tag. Can you check what the dropdown shows underneath the ontology name in the list on the left? If it says "Tag: ___", then you are currently viewing the version of the ontology at that Tag. Tags are immutable pointers to specific commits, hence why you cannot make any changes. To get back to an editable state, select a branch from that dropdown list.

In the second case, it sounds like the ontology you uploaded had imported entities. Imported entities cannot be edited through our tool as we suggest that imported ontologies should be used as references, not changed.

**Comment by gregid** [comment]
Can't check first case as I have already deleted it (but don't recall adding any tags) and recreated anew but I confirm that is the behaviour when I select a tag on another ontology.

Second - that would explain the behaviour I see. The ontology I uploaded is http://semanticarts.com/gist
When I upload gistCore (aggregated individual ontologies) all the fields are greyed out, when I do it for individual files, e.g. gistPlace I can edit entities that belong to Place (with rest greyed out). I will play with it a bit to see if mobi's approach to imported ontologies cause any issues (but I see the reasoning behind it).

Thank you for a great tool and your assistance
